src/bootstrap/src/lib.rs

```diff
@@ -655,27 +655,37 @@ impl Build {
     }
 
     /// Gets the space-separated set of activated features for the standard
-    /// library.
+    /// library. This can be configured with the `std_features` key in config.toml.
     fn std_features(&self, target: TargetSelection) -> String {
-        let mut features = " panic-unwind".to_string();
+        let mut features = if let Some(features) = &self.config.rust_std_features {
+            features.iter().map(|s| s.as_ref()).collect::<Vec<&str>>()
+        } else {
+            vec![]
+        };
+
+        if !features.contains(&"panic-unwind") {
+            features.push("panic-unwind");
+        }
```

Comment:
```suggestion
        features.push("panic-unwind");
```

This works too, right?